packages/p2p-v2/src/components/BuySellForm/__tests__/BuySellForm.spec.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { PropsWithChildren } from 'react';
+import Modal from 'react-modal';
+import { floatingPointValidator } from '@/utils';
+import { useDevice } from '@deriv-com/ui';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import BuySellForm from '../BuySellForm';
+
+const mockMutateFn = jest.fn();
+jest.mock('@deriv/api', () => ({
+    p2p: {
+        order: {
+            useCreate: jest.fn(() => ({
+                mutate: mockMutateFn,
+            })),
+        },
+    },
+}));
+
+jest.mock('@deriv-com/ui', () => ({
+    ...jest.requireActual('@deriv-com/ui'),
+    useDevice: jest.fn(() => ({ isMobile: false })),
+}));
+
+const mockUseDevice = useDevice as jest.Mock;
+const mockOnChange = jest.fn();
+jest.mock('react-hook-form', () => ({
+    ...jest.requireActual('react-hook-form'),
+    Controller: ({ control, defaultValue, name, render }) =>
+        render({
+            field: { control, name, onBlur: jest.fn(), onChange: mockOnChange, value: defaultValue },
+            fieldState: { error: null },
+        }),
+    useForm: () => ({
+        control: 'mockedControl',
+        formState: { isValid: true },
+        getValues: jest.fn(() => ({
+            amount: 1,
+        })),
+        handleSubmit: jest.fn(),
+    }),
+}));
+
+jest.mock('@/utils', () => ({
+    ...jest.requireActual('@/utils'),
+    floatingPointValidator: jest.fn(() => false),
+}));
+const mockFloatingPointValidator = floatingPointValidator as jest.Mock;
+
+const wrapper = ({ children }: PropsWithChildren<unknown>) => <div id='v2_modal_root'>{children}</div>;
+
+const mockAdvertValues = {
+    account_currency: 'USD',
+    advertiser_details: {
+        name: 'name',
+        rating_average: 5,
+        rating_count: 5,
+    },
+    description: 'description',
+    id: 'id',
+    is_buy: true,
+    local_currency: 'USD',
+    max_order_amount_limit_display: '1000',
+    min_order_amount_limit: 1,
+    min_order_amount_limit_display: '1',
+    order_expiry_period: 30,
+    payment_method_names: ['alipay'],
+    rate: 1,
+    rate_type: 'fixed',
+    type: 'sell',
+};
+const mockProps = {
+    advert: mockAdvertValues,
+    advertiserBuyLimit: 1000,
+    advertiserSellLimit: 1000,
+    balanceAvailable: 10,
+    displayEffectiveRate: '1',
+    effectiveRate: 1,
+    isModalOpen: true,
+    onRequestClose: jest.fn(),
+    paymentMethods: [
+        {
+            display_name: 'alipay',
+            type: 'online',
+        },
+    ],
+};
+let element: HTMLElement;
+describe('BuySellForm', () => {
+    beforeAll(() => {
+        element = document.createElement('div');
```

Comment:
@nada-deriv, instead of using `document.createElement` here, is it possible to retrieve the already existing `div` with the `v2_modal_root` id that was defined [here](https://github.com/binary-com/deriv-app/pull/13747/files#:~:text=const%20wrapper%20%3D%20(%7B%20children%20%7D%3A%20PropsWithChildren%3Cunknown%3E)%20%3D%3E%20%3Cdiv%20id%3D%27v2_modal_root%27%3E%7Bchildren%7D%3C/div%3E%3B)
```suggestion
        element = document.createElement('div');
```